nuxt.config.ts

```diff
@@ -76,6 +76,7 @@ export default defineNuxtConfig({
       walletConnectProjectId: process.env.WALLET_CONNECT_PROJECT_ID,
       arweaveEndpoint: process.env.ARWEAVE_ENDPOINT,
       ipfsEndpoint: process.env.IPFS_ENDPOINT,
+      isMaintenanceMode: process.env.IS_MAINTENANCE_MODE === 'TRUE',
```

Comment:
```suggestion
      isMaintenanceMode: process.env.IS_MAINTENANCE_MODE,
```

Reply:
Updated with `!== undefined` for boolean type